scripts/ingests/Gaia/ingest_gaia.py

```diff
@@ -0,0 +1,362 @@
+VERBOSE = True
+DATE_SUFFIX = 'Sep2021'
+
+verboseprint = print if VERBOSE else lambda *a, **k: None
+
+db = load_simpledb(RECREATE_DB=RECREATE_DB)
+
+# get all sources that have Gaia DR2 designations
+# 2nd query which is all sources that are not in that list
+gaia_sources = db.query(db.Names.c.source).filter(db.Names.c.other_name.ilike("Gaia DR2%")).table()
+if gaia_sources:
+    gaia_sources_list = gaia_sources['source'].tolist()
+    no_gaia_source_id = db.query(db.Sources.c.source).filter(db.Sources.c.source.notin_(gaia_sources_list)).\
+        table()['source'].tolist()
+else:
+    no_gaia_source_id = db.query(db.Sources.c.source).table()['source'].tolist()
+
+
+
+def find_in_simbad(no_gaia_source_id, desig_prefix, source_id_index = None, verbose = False):
+    """
+    Function to extract source designations from SIMBAD
+
+    Parameters
+    ----------
+    no_gaia_source_id
+    desig_prefix
+    source_id_index
+    verbose
+
+    Returns
+    -------
+    Astropy table
+
+    """
+
+    verboseprint = print if VERBOSE else lambda *a, **k: None
+
+    sources = no_gaia_source_id
+    n_sources = len(sources)
+
+    Simbad.reset_votable_fields()
+    Simbad.add_votable_fields('typed_id')  # keep search term in result table
+    Simbad.add_votable_fields('ids')  # add all SIMBAD identifiers as an output column
+    print("simbad query started")
+    result_table = Simbad.query_objects(sources)
+    print("simbad query ended")
+
+    ind = result_table['SCRIPT_NUMBER_ID'] > 0  # find indexes which contain results
+
+    simbad_ids = result_table['TYPED_ID', 'IDS'][ind]  # .topandas()
+
+    db_names = []
+    simbad_designations = []
+    if source_id_index is not None:
+        source_ids = []
+
+    for row in simbad_ids:
+        db_name = row['TYPED_ID']
+        ids = row['IDS'].split('|')
+        designation = [i for i in ids if desig_prefix in i]
+
+        if designation:
+            verboseprint(db_name, designation[0])
+            db_names.append(db_name)
+            simbad_designations.append(designation[0])
+            if source_id_index is not None:
+                source_id = designation[0].split()[source_id_index]
+                source_ids.append(int(source_id)) #convert to int since long in Gaia
+
+    n_matches = len(db_names)
+    print('Found', n_matches, desig_prefix, ' sources for', n_sources, ' sources')
+
+    result_table = Table([db_names, simbad_designations, source_ids],
+                      names=('db_names', 'designation', 'source_id'))
+
+    return result_table
+
+
+# Use SIMBAD to find Gaia DR2 and DR3 designations for sources without a Gaia designation
+# dr2_designations = find_in_simbad(no_gaia_source_id, 'Gaia DR2', source_id_index=2, verbose=VERBOSE)
+dr2_desig_file_string = 'scripts/ingests/Gaia/gaia_dr2_designations_'+DATE_SUFFIX+'.xml'
+# dr2_designations.write(dr2_desig_file_string, format='votable', overwrite=True)
+dr2_designations = Table.read(dr2_desig_file_string, format='votable')
```

Comment:
In development/testing, I understand why you would want to store this to a file, but I don't think this is something that needs to be added to the repo. The Gaia designations for DR2 are not going to change if you run this today, and if this is meant to be re-run we would want to re-create it anyways since we may have more sources in the SIMPLE database. In short: because these xml are dynamically drawn from external sources I would argue that we not store the xml results in this repo and instead have the code to regenerate them (which you do here).

Reply:
I removed the XML files from the PR.